src/x11/x.c

```diff
@@ -584,15 +584,46 @@ static dimension_t x_render_layout(cairo_t *c, colored_layout *cl, colored_layou
         int bg_height = MAX(settings.notification_height, (2 * settings.padding) + h);
         double bg_half_height = settings.notification_height/2.0;
         int pango_offset = (int) floor(h/2.0);
+        double degrees = M_PI / 180.0;
 
         if (first) bg_height += settings.frame_width;
         if (last) bg_height += settings.frame_width;
         else bg_height += settings.separator_height;
 
-        cairo_set_source_rgb(c, cl->frame.r, cl->frame.g, cl->frame.b);
+        cairo_set_source_rgba(c, cl->bg.r, cl->bg.g, cl->bg.b, 0);
         cairo_rectangle(c, bg_x, bg_y, bg_width, bg_height);
         cairo_fill(c);
 
+        if (settings.frame_width > 0) {
+                if (first && last) {
+                        cairo_new_sub_path (c);
+                        cairo_arc (c, bg_x + bg_width - settings.corner_radius, bg_y + settings.corner_radius, settings.corner_radius, -90 * degrees, 0 * degrees);
```

Comment:
Another style nitpick but I think this is borderline unreadable. I'd prefer if it was one parameter per line for long function calls like so:
```c
                cairo_arc (c,
                           bg_x + bg_width - corner_radius,
                           bg_y + corner_radius,
                           corner_radius,
                           -90 * degrees,
                           0 * degrees);
```

Same goes for `XFillArc` and `XFillRectangle`. Though I'm not 100% on this decision. @bebehei What do you think?

(I'm aware that this is the case for other calls in the same function but since it came up now might as-well discuss it and maybe start fixing it.)

Reply:
>  @bebehei What do you think?

Well, I'd like to have parameters in one line. I prefer it.